tests/utils/SparseGridTest.cpp

```diff
@@ -0,0 +1,270 @@
+//Copyright (c) 2015 Ultimaker B.V.
```

Comment:
This diff went horribly wrong :(

It didn't see it as a rename of the file.

I suppose this is just a simple refactor, so I won't review it.